openmoc/__init__.py

```diff
@@ -32,6 +32,7 @@
 time = (now.month, now.day, now.year, now.hour, now.minute, now.second)
 year_string = '-'.join(map(left_pad, map(str, (now.month, now.day, now.year))))
 today_string = ':'.join(map(left_pad, map(str, (now.hour, now.minute, now.second))))
+today_string += ':' + str(now.microsecond)[:2]
```

Comment:
That's precisely why I added the sub-second resolution to the log filename here.